### PR TITLE
i18n: add pt-BR translations

### PR DESCRIPTION
**Pull request by vctt94**
This branch adds pt-br into dex.

Depends on #1127

### CODE REVIEW

File: client/webserver/site/src/js/locales.js [hunk 1]
```diff
@@ -64,8 +64,41 @@ export const templateKeys = {
...
+  [ID_NO_ZERO_RATE]: 'taxa não pode ser zero',
+  [ID_NO_ZERO_QUANTITY]: 'quantidade não pode ser zero',
+  [ID_TRADE]: 'trade',
+  [ID_NO_ASSET_WALLET]: 'Sem carteira {{ asset }}',
+  [ID_EXECUTED]: 'executado',
+  [ID_BOOKED]: 'reservado',
+  [ID_CANCELING]: 'cancelando',
+  [ID_ACCT_UNDEFINED]: 'conta não definida.',
+  [ID_KEEP_WALLET_PASS]: 'manter senha da carteira',
+  [ID_NEW_WALLET_PASS]: 'definir nova senha para carteira'
```
Comment (chappjc) [inline client/webserver/site/src/js/locales.js:96]: Missing the added `ID_LOT` and `ID_LOTS`.

File: client/webserver/locales/pt-br.go [hunk 2]
```diff
@@ -0,0 +1,171 @@
+package locales
+
+var PtBr = map[string]string{
+	"Language":              "pt-BR",
+	"Markets":               "Mercados",
+	"Wallets":               "Carteiras",
```
Comment (chappjc) [inline client/webserver/locales/pt-br.go:6]: Would you add a map entry for `Export Trades` please.

https://github.com/decred/dcrdex/blob/6e6cac4bf16f42ee6ad9e47721bb15263874f22d/client/webserver/site/src/html/orders.tmpl#L39

**Review by buck54321 — CHANGES_REQUESTED** [review]
While reviewing this, I found some missing translations and added them in #1200. I hope we can sneak that in first and add those translations here.

But this work looks good. This PR will be a good reference for the next translator. And it's super fun using DEX in Portugese.

**Review by buck54321 — APPROVED** [review]
Good to go. 

I did find a couple more stray strings that weren't being translated, if you wanted to pick them up here.

`'Waiting for confirmations...'` (& others in `updateRegistrationStatusView`)
`'none selected'`

**Review by chappjc — APPROVED** [review]
LGTM too.  @vctt94 just have a look at buck's final review when you get a chance to make some small tweaks.

**Pull request merged**